packages/contracts-bedrock/test/L2/L2StandardBridge.t.sol

```diff
@@ -389,17 +417,50 @@ contract L2StandardBridge_WithdrawTo_Test is L2StandardBridge_TestInit {
 
     /// @notice Tests that `withdrawTo` burns the tokens, emits `WithdrawalInitiated`, and
     ///         initiates a withdrawal with `Withdrawer.initiateWithdrawal`.
-    function test_withdrawTo_withdrawingERC20_succeeds() external {
-        _preBridgeERC20To({ _isLegacy: true, _l2Token: address(L2Token) });
-        l2StandardBridge.withdrawTo(address(L2Token), bob, 100, 1000, hex"");
+    function testFuzz_withdrawTo_withdrawingERC20_succeeds(uint256 _amount) external {
+        _amount = bound(_amount, 1, 1000000);
+
+        deal(address(L2Token), alice, _amount, true);
+        assertEq(L2Token.balanceOf(alice), _amount);
+
+        vm.expectEmit(address(l2StandardBridge));
+        emit WithdrawalInitiated({
+            l1Token: address(L1Token),
+            l2Token: address(L2Token),
+            from: alice,
+            to: bob,
+            amount: _amount,
+            data: hex""
+        });
+
+        vm.expectEmit(address(l2StandardBridge));
+        emit ERC20BridgeInitiated({
+            localToken: address(L2Token),
+            remoteToken: address(L1Token),
+            from: alice,
+            to: bob,
+            amount: _amount,
+            data: hex""
+        });
+
+        vm.prank(alice, alice);
+        l2StandardBridge.withdrawTo(address(L2Token), bob, _amount, 1000, hex"");
```

Comment:
The `1000` here is `minGasLimit`, and should be fuzzed as we did for `testFuzz_withdraw_ether_succeeds`.